client/src/services/notificationCenter.ts

```diff
@@ -43,6 +49,15 @@ export class Notification {
   }
 }
 
+export interface NotificationToastConfig {
```

Comment:
Not much point to this interface. It seems it's only used once when calling `_createAndPresentToast`, might as well just pass arguments to the function, no need for an interface.

Reply:
export was useless.
However the interface was helping to type the function easily. Now we're forced to: 
```
    const toastConfig: any = {
      theme: this._getMsTheme(notification.level),
      icon: this._getToastIcon(notification.level),
      message: notification.message,
    };
    if (notification.title) {
      toastConfig.title = notification.title;
    }
```